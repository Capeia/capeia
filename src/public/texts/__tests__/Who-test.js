// @flow
/* eslint-env jest, jasmine */

import React from 'react'
import renderer from 'react-test-renderer'

describe('<Who>', () => {
  it('renders correctly', () => {
    jest.deepUnmock('public/texts/Who')
    const Component = require('public/texts/Who').default
    const tree = renderer.create(<Component />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
