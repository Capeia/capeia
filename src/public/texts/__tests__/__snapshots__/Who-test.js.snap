// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Who> renders correctly 1`] = `
<div
  className="container-fluid"
>
  <div
    className="IntroContainer row"
  >
    <div
      className="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1"
    >
      <div
        className=""
      >
        <h1>
          Who
        </h1>
        <div
          className="row"
          style={
            Object {
              "marginBottom": "2em",
            }
          }
        >
          <div
            className="col-md-3"
            style={
              Object {
                "textAlign": "center",
              }
            }
          >
            <img
              src="who/portrait_rs.jpg"
              style={
                Object {
                  "height": "auto",
                  "width": 200,
                }
              }
            />
          </div>
          <div
            className="col-md-9"
          >
            <p>
              RÃ¼diger Schweigreiter is a biologist with a PhD in neuroscience. He lives and works in both Innsbruck and Vienna. Born and raised in the Alps, he spends as much time as possible in Australia. Bushwalking in the Blue Mountains or spending a night in Tasmania\`s Central Highlands provide memorable impressions that he would not want to miss.
            </p>
          </div>
        </div>
        <div
          className="row"
        >
          <div
            className="col-md-3 col-md-push-9"
            style={
              Object {
                "textAlign": "center",
              }
            }
          >
            <img
              src="who/portrait_ps.jpg"
              style={
                Object {
                  "height": "auto",
                  "width": 200,
                }
              }
            />
          </div>
          <div
            className="col-md-9 col-md-pull-3"
          >
            <p>
              Philip Salzmann is a software engineer and web designer based in Austria. He believes that education is the most valuable resource of the present. Fascinated by science since his early childhood days, he looks forward to read about new developments in all areas on Capeia.com!
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
